Competition2018/src/main/java/competition/subsystems/elevator/commands/TrackCurrentUsageCommand.java

```diff
@@ -0,0 +1,61 @@
+package competition.subsystems.elevator.commands;
+
+import java.util.HashMap;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseCommand;
+import xbot.common.controls.actuators.XCANTalon;
+
+public class TrackCurrentUsageCommand extends BaseCommand {
+
+    XCANTalon talon;
+    double current;
+    double time;
+    double currentAveragingWindow;
+
+    HashMap<Double, Double> currentHistory;
+    
+    @Inject
+    public TrackCurrentUsageCommand(XCANTalon talon) {
+        this.talon = talon;
+        currentHistory = new HashMap<Double, Double>();
+        currentAveragingWindow = 1000;
```

Comment:
Woah! That's big number. Perhaps something <100 would be better; I'd suggest more like 50 or 25. If you want samples over a longer period of time, I'd suggest only adding a sample every, say, half-second.

Reply:
did it!